.vsts-ci/templates/test/nix-test-steps.yml

```diff
@@ -0,0 +1,66 @@
+parameters:
+  purpose: ''
+  tagSet: 'CI'
+
+steps:
+  - pwsh: |
+      Get-ChildItem -Path env:
+    displayName: Capture Environment
+    condition: succeededOrFailed()
+
+  - task: DownloadBuildArtifacts@0
+    displayName: 'Download build artifacts'
+    inputs:
+      downloadType: specific
+      itemPattern: |
+        build/**/*
+      downloadPath: '$(System.ArtifactsDirectory)'
+
+  - pwsh: |
+      Get-ChildItem "$(System.ArtifactsDirectory)\*" -Recurse
+    displayName: 'Capture Artifacts Directory'
+    continueOnError: true
+
+  - pwsh: |
+      if (Get-Command tdnf -errorAction SilentlyContinue) {
+        sudo tdnf install -y unzip git
+      }
+    displayName: 'Install unzip, git using tdnf'
+    continueOnError: true
+
+  - pwsh: |
+      Import-Module .\tools\ci.psm1
+      Invoke-CIInstall -SkipUser
+    displayName: Bootstrap
+
+  - task: ExtractFiles@1
+    displayName: 'Extract Build ZIP'
+    inputs:
+      archiveFilePatterns: '$(System.ArtifactsDirectory)/build/build.zip'
+      destinationFolder: '$(System.ArtifactsDirectory)/bins'
+
+  - bash: |
+      find "$(System.ArtifactsDirectory)/bins" -type d -exec chmod +rwx {} \;
+      find "$(System.ArtifactsDirectory)/bins" -type f -exec chmod +rw {} \;
```

Comment:
i would have thought we would want to force the files to be `rwx` to be executable. There might be some issues with tests which write into this directory tree in a misguided attempt to get write errors.

Reply:
Line 61 sets the file that needs to be executable, as executable, this is just generically setting base permissions.

If we want to remove write permissions from the directory and files, that should be another PR.